ZupIT/beagle-web-angular
# Fix: Avoid unnecessary logic when no Data Source is provided on Dynamic List

**- What I did**
Avoid unnecessary logic when no Data Source is provided on Dynamic List.
Also increased the version of the package to stop the warnings on the playground.

**- How I did it**
Adding a simple condition to the dataSource if at the beginning of the function.

**- How to verify it**
Create a JSON with the dataSource empty and run it, the same rule applies for asynchronous DataSource, it should not throw any warning or message.
